core/tools/blueos_startup_update/blueos_startup_update

```diff
@@ -28,6 +28,28 @@ DELTA_JSON = {
 
 import collections
 
+import subprocess
+
+def run_command_in_host(command: str, check: bool = True) -> "subprocess.CompletedProcess['str']":
```

Comment:
Since this is already used in `commander`, we may want to move it to `commonwealth`, avoid dupplicating the code.

Reply:
done